nix/default.nix

```diff
@@ -178,6 +178,7 @@ in rec {
     jq = bazel_dependencies.jq;
     sed = pkgs.gnused;
     sha1sum = pkgs.coreutils;
+    sha256sum = pkgs.coreutils;
```

Comment:
That makes a ton of sense as a reason for things not to work, but it's very confusing as a reason for things to _stop working now_.

Reply:
yeah I don’t understand it but given that this fixes it in my tests, I’m also not that keen in spending more time debugging it. Maybe someone updated the macos builders or osmething like that.

Reply:
I'll run a compat test on this commit to be sure.